Notifications: Project already disposed

As reported via Gitter by @mogren , notifications for opening settings pages might cause errors if project is disposed.

```
Already disposed: Project (Disposed) via-mw-adapter-android
java.lang.AssertionError: Already disposed: Project (Disposed) via-mw-adapter-android
    at com.intellij.openapi.components.impl.ComponentManagerImpl.getPicoContainer(ComponentManagerImpl.java:265)
    at com.intellij.openapi.components.impl.ComponentManagerImpl.getPicoContainer(ComponentManagerImpl.java:52)
    at com.intellij.openapi.components.ServiceManager.doGetService(ServiceManager.java:46)
    at com.intellij.openapi.components.ServiceManager.getService(ServiceManager.java:41)
    at com.intellij.ide.util.PropertiesComponent.getInstance(PropertiesComponent.java:82)
    at com.intellij.openapi.options.newEditor.SettingsEditor.<init>(SettingsEditor.java:72)
    at com.intellij.openapi.options.newEditor.SettingsDialog.<init>(SettingsDialog.java:70)
    at com.intellij.ide.actions.ShowSettingsUtilImpl.getDialog(ShowSettingsUtilImpl.java:57)
    at com.intellij.ide.actions.ShowSettingsUtilImpl.showSettingsDialog(ShowSettingsUtilImpl.java:123)
    at de.plushnikov.intellij.plugin.LombokPluginProjectValidatorComponent$SettingsOpeningListener.hyperlinkActivated(LombokPluginProjectValidatorComponent.java:73)
    at com.intellij.notification.NotificationListener$Adapter.hyperlinkUpdate(NotificationListener.java:31)
`
``

--- a/src/main/java/de/plushnikov/intellij/plugin/LombokPluginProjectValidatorComponent.java
+++ b/src/main/java/de/plushnikov/intellij/plugin/LombokPluginProjectValidatorComponent.java
@@ -17,13 +17,15 @@ import com.intellij.openapi.roots.ModuleRootManager;
 import com.intellij.openapi.roots.OrderEntry;
 import com.intellij.psi.JavaPsiFacade;
 import com.intellij.psi.PsiPackage;
-import de.plushnikov.intellij.plugin.settings.ProjectSettings;
+
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
-import javax.swing.event.HyperlinkEvent;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import javax.swing.event.HyperlinkEvent;
+
+import de.plushnikov.intellij.plugin.settings.ProjectSettings;
 
 /**
  * Shows notifications about project setup issues, that make the plugin not working.
@@ -175,7 +177,10 @@ public class LombokPluginProjectValidatorComponent extends AbstractProjectCompon
 
     @Override
     protected void hyperlinkActivated(@NotNull final Notification notification, @NotNull final HyperlinkEvent e) {
-      ShowSettingsUtil.getInstance().showSettingsDialog(project, nameToSelect);
+
+      if (!project.isDisposed()) {
+        ShowSettingsUtil.getInstance().showSettingsDialog(project, nameToSelect);
+      }
     }
   }
 }

